templates/CRM/common/paymentBlock.tpl

```diff
@@ -80,6 +80,10 @@
     }
     showHidePayment(isHide);
   }
+  // If we get a validation error and the form reloads we can end up with this still showing the selected pp
+  //  but the billing section is for the first payproc.
+  CRM.$('#payment_processor_id > option:selected').each(function () {CRM.$(this).removeAttr('selected')});
```

Comment:
You are deselecting every option in a select field? It's not a multiselect is it?

Reply:
@colemanw No it's a standard select... which is different from frontend forms which use a set of radios, which is why this is not working on the backend.
![localhost_8000_civicrm_contact_view_reset 1 cid 203](https://user-images.githubusercontent.com/2052161/33573841-0b610ac4-d92f-11e7-96f1-a311c67f319c.png)